### DISCUSSION TITLE
How to load local json options?

### DISCUSSION THREAD

**Discussion by ccasado16** [matteobruni/tsparticles#4939] [Q&A] [answered]
I'm trying to load the options from a local json file but i can't?
Could u help me?

Here's my options.json
{
    "background": {
      "color": {
        "value": "#0d47a1"
      }
    },
    "backgroundMode": {
      "enable": true,
      "zIndex": -1
    },
    "fpsLimit": 120,
    "interactivity": {
      "events": {
        "onClick": {
          "enable": true,
          "mode": "push"
        },
        "onHover": {
          "enable": true,
          "mode": "grab"
        },
        "resize": true
      },
      "modes": {
        "bubble": {
          "distance": 400,
          "duration": 2,
          "opacity": 0.8,
          "size": 40
        },
        "push": {
          "quantity": 4
        },
        "repulse": {
          "distance": 200,
          "duration": 0.4
        }
      }
    },
    "particles": {
      "color": {
        "value": "#ffffff"
      },
      "links": {
        "color": "#ffffff",
        "distance": 150,
        "enable": true,
        "opacity": 0.5,
        "width": 1
      },
      "collisions": {
        "enable": false
      },
      "move": {
        "direction": "none",
        "enable": true,
        "outMode": "bounce",
        "random": false,
        "speed": 3,
        "straight": false
      },
      "number": {
        "density": {
          "enable": true,
          "area": 800
        },
        "value": 80
      },
      "opacity": {
        "value": 0.5
      },
      "shape": {
        "type": "circle"
      },
      "size": {
        "random": true,
        "value": 5
      }
    },
    "detectRetina": true
}

Here's what i'm trying to get the options loaded:
![code](https://user-images.githubusercontent.com/100170200/230663178-a255d3f5-b5ce-4d4a-9383-0d500e1b24e0.png)

**Comment by matteobruni** [comment, marked as answer]
Remove the brackets from options and it should work. I’ll close this, if you need more help on this please open an issue in the right repository https://github.com/tsparticles (choose the right repository then open an issue there)
